accord-core/src/test/java/accord/burn/random/FrequentLargeRangeTest.java

```diff
@@ -30,7 +30,7 @@
 
 import static accord.utils.Property.qt;
 
-class SegmentedRandomRangeTest
+class FrequentLargeRangeTest
```

Comment:
looks like I forgot to rename this class when I renamed `SegmentedRandomRange ` to `FrequentLargeRange`